test/Microsoft.Azure.Devices.Edge.Test.Common/IotHub.cs

```diff
@@ -232,6 +234,20 @@ public async Task<Device> CreateEdgeDeviceIdentityAsync(string deviceId, Option<
             await receiver.CloseAsync();
         }
 
+        static async Task<EventHubRuntimeInformation> GetPartitionCountAsync(EventHubClient client, CancellationToken token)
+        {
+            CancellationToken eachRequestCancellationToken = new CancellationTokenSource(eventHubRequestDuration).Token;
+
+            // Sometimes eventhub will hang when getting runtime information, so we need a retry.
+            var retryStrategy = new Incremental(15, RetryStrategy.DefaultRetryInterval, RetryStrategy.DefaultRetryIncrement);
+            var retryPolicy = new RetryPolicy(new CatchTimeoutErrorDetectionStrategy(), retryStrategy);
+            return await retryPolicy.ExecuteAsync(
+                async () =>
+            {
+                return await Task.Run(async () => await client.GetRuntimeInformationAsync(), eachRequestCancellationToken);
+            }, token);
+        }
```

Comment:
FYA, added this retry

Reply:
If this does not happen again, My preference would be to remove the retry.

Reply:
It happens in the pipeline sometimes so I think we need to keep the retry.